index.js

```diff
@@ -26,22 +29,25 @@ function JMON(data) {
   });
 }
 
-
-JMON.prototype.isCreated = false;
-JMON.prototype.isUpdated = false;
-JMON.prototype.isDeleted = false;
-
-
 /**
  * This will commit current changes and treat current data as initial data
  * Also clears, createdList, deletedList, updatedList
  */
 JMON.prototype.commit = function() {
 
   this.initialData = _.cloneDeep(this.data);
-  Object.getPrototypeOf(this).isCreated = false;
-  Object.getPrototypeOf(this).isUpdated = false;
-  Object.getPrototypeOf(this).isDeleted = false;
+  this.isCreated = false;
+  this.isUpdated = false;
+  this.isDeleted = false;
+};
+
+/**
+ * Returns true if self or any of children has been modified.
+ * Returns false if nothing has been changed.
+ */
+JMON.prototype.checkIfUpdated = function() {
```

Comment:
I think we should name this function to `isModified`. That will be better. `checkIfUpdated` sounds a bit odd.
Any inputs on this?

Reply:
ya  that sounds better.